README.md

```diff
@@ -46,56 +57,46 @@ class SquareToCircle(Scene):
         self.play(FadeOut(square))
 ```
 
-Save this code in a file called `example.py`. Now open your terminal in the
-folder where you saved the file and execute
+In order to view the output of this scene, save the code in a file called `example.py`. Then, run the following in a terminal window:
 
 ```sh
-manim example.py SquareToCircle -pl
+manim example.py SquareToCircle -p -ql
 ```
 
-You should see your video player pop up and play a simple scene where a square
-is transformed into a circle. You can find some more simple examples in the
-[GitHub repository](https://github.com/ManimCommunity/manim/tree/master/example_scenes).
-Visit the [official gallery](https://manimce.readthedocs.io/en/latest/examples.html) for more advanced examples.
+You should see your native video player program pop up and play a simple scene in which a square is transformed into a circle. You may find some more simple examples within this
+[GitHub repository](https://github.com/ManimCommunity/manim/tree/master/example_scenes). You can also visit the [official gallery](https://manimce.readthedocs.io/en/latest/examples.html) for more advanced examples.
 
 ## Command line arguments
 
-The general usage of manim is as follows:
+The general usage of `manim` is as follows:
 
 ![manim-illustration](https://raw.githubusercontent.com/ManimCommunity/manim/master/readme-assets/command.png)
 
 The `-p` flag in the command above is for previewing, meaning the video file will automatically open when it is done rendering. The `-l` flag is for a faster rendering at a lower quality.
 
 Some other useful flags include:
 
-- `-s` to skip to the end and just show the final frame.
-- `-n <number>` to skip ahead to the `n`'th animation of a scene.
-- `-f` show the file in the file browser.
+-  `-s` to skip to the end and just show the final frame.
+-  `-n <number>` to skip ahead to the `n`'th animation of a scene.
+-  `-f` show the file in the file browser.
 
-For a thorough list of command line arguments, visit the
-[documentation](https://manimce.readthedocs.io/en/latest/tutorials/configuration.html).
+For a thorough list of command line arguments, visit the [documentation](https://manimce.readthedocs.io/en/latest/tutorials/configuration.html).
 
 ## Documentation
 
 Documentation is in progress at [ReadTheDocs](https://manimce.readthedocs.io/en/latest/).
 
 ## Help with Manim
 
-If you need help installing or using Manim, please take a look at [the Reddit
-Community](https://www.reddit.com/r/manim) or the [Discord
-Community](https://discord.gg/mMRrZQW). For bug reports and feature requests,
-please open an issue.
+If you need help installing or using Manim, feel free to reach out to our [Reddit Community](https://www.reddit.com/r/manim) or [Discord
+Server](https://discord.gg/mMRrZQW). If you would like to submit bug report or feature request, please open an issue.
```

Comment:
It'd be better to place the discord before the subreddit, since it's more active.